16x9/DialogFullScreenInfo.xml

```diff
@@ -698,64 +702,71 @@
 		<!-- LiveTV Plot-->
 		<control type="group" id="9006">
 			<top>10</top>
-			<animation effect="slide" end="0,100" time="240" tween="quadratic" condition="Window.IsActive(videoosd)">Conditional</animation>
-			<animation effect="fade" start="0" end="100" time="400">WindowOpen</animation>
-			<animation effect="fade" start="100" end="0" time="400">WindowClose</animation>
```

Comment:
The animation differs from the Infobar animation which I didn't like. The infobar is gone before the plotinfo fades out.
Maybe we can add this (or another) animation to both LiveTV-plotinfo and LiveTV-infobar